src/Dfe.PlanTech.Infrastructure.Data/PlanTechDbContext.cs

```diff
@@ -132,4 +132,32 @@ public Task<int> ExecuteRaw(FormattableString sql, CancellationToken cancellatio
     public Task<T?> FirstOrDefaultAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default) => queryable.FirstOrDefaultAsync(cancellationToken);
 
     public Task<List<T>> ToListAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default) => queryable.ToListAsync(cancellationToken);
+
+    public async Task<List<Answer>> GetAnswersForLatestSubmissionBySectionId(string sectionId, int establishmentId)
+    {
+        var latestSubmissionIdQuery = Submissions
+            .Where(s => s.SectionId == sectionId && s.EstablishmentId == establishmentId && s.Completed)
+            .OrderByDescending(s => s.DateCreated)
+            .Select(s => s.Id)
+            .FirstOrDefaultAsync();
+
+        if (latestSubmissionIdQuery == null)
```

Comment:
This is never going to be null, as we haven't executed the query yet. So `latestSubmissionIdQuery` is quite literally the query; not the result. You'll need to check `latestSubmissionId` for null.